docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -0,0 +1,253 @@
+1. Right-click your app service and select **Open in Portal**.
+1. When the portal opens in the browser, copy the URL of the web site from the **Overview** pane and save it. You'll need it in later steps.
+
+## Update manifest
+
+It's useful to maintain multiple manifests for testing across localhost, staging, and deployment. We recommend you copy the existing file and create a new manifest named **manifest-deployment.xml**.
+
+1. Open the **manifest-deployment.xml** file.
+1. Find all instances of the text `localhost:3000` and replace it with the domain of your app service URL you saved previously.
+1. In the `<AppDomains>` section, add an `<AppDomain>` entry for your app service from the URL you saved previously. For example `<AppDomain>https://contoso-sso.azurewebsites.net</AppDomain>`.
+1. Save the file.
+
+## Update package.json
+
+1. In the Visual Studio Code terminal, run the following command to modify the package to start the Node JS server when deployed.
+
+    ```command
+    `npm pkg set start="node middletier.js"`.
+    ```
+
+1. Run the following command to remove the `prestart` script which is not needed.
+
+    ```command
+    `npm pkg delete scripts.prestart`
+    ```
+
+## Update webpack.config.js
+
+1. Open the **webpack.config.js** file.
+1. Find the first `CopyWebpackPlugin` section and update it to also copy the package.json file to the dist folder as shown in the following example.
+
+   ```javascript
+    new CopyWebpackPlugin({
+          patterns: [
+            {
+              from: "assets/*",
+              to: "assets/[name][ext][query]",
+            },
+            {
+              from: "package.json",
+              to: "package.json",
+            },
+            {
+              from: "manifest*.xml",
+              to: "[name]" + "[ext]",
+              transform(content) {
+                if (dev) {
+                  return content;
+                } else {
+                  return content.toString().replace(new RegExp(urlDev, "g"), urlProd);
+                }
+              },
+            },
+          ],
+        }),
+   ```
+
+1. Save the file.
+
+## Update fallbackauthdialog.js (or fallbackauthdialog.ts)
+
+1. Open the **src/helpers/fallbackauthdialog.js** file, or **src/helpers/fallbackauthdialog.ts** if your project uses TypeScript.
+1. Find the `redirectUri` on line 24 and change the value to use your app service URL you saved previously. For example, `redirectUri: "https://contoso-sso.azurewebsites.net/fallbackauthdialog.html",`
+1. Save the file.
+
+## Update .ENV
+
+The **.ENV** file contains a client secret. For the purposes of learning in this article you can deploy the **.ENV** file to Azure. However for a production deployment, you should move the secret and any other confidential data into [Azure Key Vault](/azure/key-vault/general/basic-concepts).
+
+1. Open the **.ENV** file.
+1. Set the `NODE_ENV` variable to the value `production`.
+1. Save the file.
+
+## Update app.js (or app.ts)
```

Comment:
Why are we doing this step? The other steps have justification in the article.

Reply:
Added justification.